Freshli/LibYearResult.cs

```diff
@@ -43,14 +46,14 @@ bool skipped
 
     public override string ToString() {
       var writer = new StringWriter();
-      writer.Write("{ _packagesValues: [ ");
+      writer.Write("{ _packagesValues: [\n");
```

Comment:
I think calling `WriteLine` would be better than using `\n` because that line separator is platform-specific.